lib/build/recipe/session/cookieAndHeaders.js

```diff
@@ -184,17 +184,11 @@ exports.getAuthModeFromHeader = getAuthModeFromHeader;
  */
 function clearSessionCookiesFromOlderCookieDomain({ req, res, config, userContext }) {
     let didClearCookies = false;
-    const cookieString = req.getHeaderValue("cookie");
-    if (cookieString === undefined || config.olderCookieDomain === undefined) {
```

Comment:
How does this handle if `olderCookieDomain` is not defined? (As far as I can see that check has been removed and we just set the cookie with an undefined domain?)

Reply:
Good catch. Missed this while refactoring. 

We will be returning a 500 response from the refreshPOST API, but yeah this function should return early if `olderCookieDomain` is undefined.